use-package.el

```diff
@@ -168,7 +168,7 @@ convert it to a string and return that."
 (defun use-package-load-name (name &optional noerror)
   "Return a form which will load or require NAME depending on
 whether it's a string or symbol."
-  (if (stringp name) `(load ,name 'noerror) `(require ',name nil 'noerror)))
```

Comment:
The error goes into `use-package-require-error-handler`. So we need to omit 'noerror param here.